.github/workflows/kics.yml

```diff
@@ -0,0 +1,38 @@
+name: kics
+
+on:
+  pull_request:
+    branches: [master]
+    paths:
+      - 'docker-compose/**'
+      - 'helm/**'
+      - '.github/workflows/kics.yml'
+  push:
+    branches: [master]
+    paths:
+      - 'docker-compose/**'
+      - 'helm/**'
+      - '.github/workflows/kics.yml'
+
+permissions:
+  security-events: write
+
+jobs:
+  kics:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - name: run kics Scan
+        uses: checkmarx/kics-github-action@530ac1f8efe6202b0f12c9a6e952597ae707b755 # v2.1.2
+        with:
+          path: 'docker-compose,helm'
+          ignore_on_exit: results
+          output_path: report-dir/
+          output_formats: 'sarif'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          enable_jobs_summary: true
+          disable_secrets: true
```

Comment:
its not possible to pick helm from the list of platform type, its there and it is scanning for it but under 
`Platform: Common` which is enabled by default in all platforms but again `Common` cannot be specified in platform type. Given that decided to not specify anything in `platform_type`. 

```
Error: unknown argument(s) for --type: helm, common
valid arguments:
  Ansible
  AzureResourceManager
  Bicep
  Buildah
  CICD
  CloudFormation
  Crossplane
  DockerCompose
  Dockerfile
  GRPC
  GoogleDeploymentManager
  Knative
  Kubernetes
  OpenAPI
  Pulumi
  ServerlessFW
  Terraform
```